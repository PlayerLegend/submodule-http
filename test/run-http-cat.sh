#!/bin/sh

echo "This test requires the ipfs daemon to be running"

cid_asdf="$(echo asdf | ipfs add --pin=false -q)"
cid_bcle="$(echo bcle | ipfs add --pin=false -q)"
cid_1234="$(echo 1234 | ipfs add --pin=false -q)"

bin/http-cat 127.0.0.1 8080 ipfs/"$cid_asdf" ipfs/"$cid_bcle" ipfs/"$cid_1234" ipfs/"$cid_1234" ipfs/"$cid_bcle" ipfs/"$cid_asdf"
