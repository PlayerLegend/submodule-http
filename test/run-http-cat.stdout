This test requires the ipfs daemon to be running
asdf
bcle
1234
1234
bcle
asdf
